src/EpisodeIntegrationService/ReceiveData/ReceiveData.cs

```diff
@@ -2,69 +2,90 @@
 using Microsoft.Azure.WebJobs;
 using Microsoft.Extensions.Logging;
 using System.Threading.Tasks;
-using Microsoft.Azure.WebJobs.Extensions.Http;
-using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.Mvc;
 using Newtonsoft.Json;
 using System.Net.Http;
 using System.Net.Http.Headers;
-using System.Collections.Generic;
 
 public static class BlobJsonTrigger
 {
-  [FunctionName("receiveData")]
-  public static async Task Run(
-      [BlobTrigger("sample-container/{name}", Connection = "AzureWebJobsStorage")] Stream myBlob,
-      string name,
-      ILogger log)
-  {
-    log.LogInformation($"C# Blob trigger function Processed blob\n Name:{name} \n Size: {myBlob.Length} Bytes");
-
-    // Validate JSON
-    using (var reader = new StreamReader(myBlob))
+    [FunctionName("ReceiveData")]
+    public static async Task Run(
+        [BlobTrigger("sample-container/{name}", Connection = "AzureWebJobsStorage")] Stream myBlob,
+        string name,
+        ILogger log)
     {
-      var jsonData = reader.ReadToEnd();
-      if (IsValidJson(jsonData))
-      {
-        log.LogInformation("JSON is valid.");
-        await SendToProcessDataFunction(jsonData, log);
-      }
-      else
-      {
-        log.LogError("JSON is invalid.");
-      }
-    }
-  }
+        log.LogInformation($"C# Blob trigger function Processed blob\n Name:{name} \n Size: {myBlob.Length} Bytes");
 
-  private static bool IsValidJson(string jsonData)
-  {
-    try
-    {
-      var obj = JsonConvert.DeserializeObject<object>(jsonData);
-      return obj != null;
+        // Validate JSON
+        using (var reader = new StreamReader(myBlob))
+        {
+            var jsonData = reader.ReadToEnd();
+            if (IsValidJson(jsonData))
+            {
+                log.LogInformation("JSON is valid.");
+                await SendToProcessDataFunction(jsonData, log);
```

Comment:
When we do post requests we should use the `SendPost` method from the `HttpRequestService` class. You'll need to create a `Program.cs` in `ReceiveData/` so that we can use Dependency Injection